src/modules/account/components/login/index.tsx

```diff
@@ -1,57 +1,83 @@
-import { useFormState } from "react-dom"
+import { useFormState } from 'react-dom';
 
-import { LOGIN_VIEW } from "@modules/account/templates/login-template"
-import Input from "@modules/common/components/input"
-import { logCustomerIn } from "@modules/account/actions"
-import ErrorMessage from "@modules/checkout/components/error-message"
-import { SubmitButton } from "@modules/checkout/components/submit-button"
+import { LOGIN_VIEW } from '@modules/account/templates/login-template';
```

Comment:
This LOGIN_VIEW should be imported from src/modules/account/components/login-modal/index.tsx.